### PR TITLE
Create sheoranharis.json

### PR DESCRIPTION
**Pull request by sheoranharis**
Created LinkFree website

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1178"></a>

### CODE REVIEW

File: sheoranharis.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            "url": "https://github.com/sheoranharis",
+            "icon": "github"
+        },
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/sheoranharis",
+            "icon": "twitter"
+        }
+      {
+            "name": "Read my Blog ",
```
Comment (Panquesito7) [inline sheoranharis.json:18]: Extra space here. Might want to remove it. 🙂
```suggestion
            "name": "Read my Blog",
```

**Review by emmalearnscode — APPROVED** [review]

File: sheoranharis.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            "url": " https://hashnode.com/@sheoranharis",
+            "icon": " Hashnode"
```
Comment (vikasganiga05) [inline sheoranharis.json:20]:
```suggestion
            "url": "https://hashnode.com/@sheoranharis",
            "icon": "Hashnode"
```
Remove these extra spaces :point_up:

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
There are some Leading spaces (at the start of the text) and Trailing spaces (at the end of the text). So, please remove these spaces.

File: sheoranharis.json (hunk 2 above)
Comment (sheoranharis) [reply sheoranharis.json]: ok,
done

**Review by krshdev — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]

File: sheoranharis.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+            "name": "Follow me on GitHub",
+            "url": "https://github.com/sheoranharis",
+            "icon": "github"
+        },
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/sheoranharis",
+            "icon": "twitter"
+        }
+      {
```
Comment (ssensalo) [inline sheoranharis.json:17]:
```suggestion
        },
        {
```

**Review by ssensalo — CHANGES_REQUESTED** [review]

File: sheoranharis.json (hunk 3 above)
Comment (Panquesito7) [reply sheoranharis.json:17]: This should be fixed automatically by the CI. 🙂

**Review by Panquesito7 — APPROVED** [review]

**Review by ssensalo — APPROVED** [review]

**Review by kabir0x23 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM!, thank you for your contribution

**Comment by sheoranharis** [comment]
I am new to the open-source world, 
I've been learning to program for more than 1 month,
can you guide me or give some advice on how to start contributing in the python language

**Review by kabir0x23 — COMMENTED** [review]
Your file is in the wrong directory.
your sheoranharis.json file must be in the Linkfree/public directory not is the (Linkfree )root directory

**Comment by Pradumnasaraf** [comment]
> Your file is in the wrong directory. your sheoranharis.json file must be in the Linkfree/public directory not is the (Linkfree )root directory

Thanks for pointing it out.

**Pull request merged**